find/main/routes.py

```diff
@@ -175,6 +178,49 @@ def retrieve_download(filename: str):
         return render_template("find/main/retrieve-download.html", context=context, form=form)
 
 
+@bp.route("/retrieve-spreadsheet/<fund_code>/<submission_id>", methods=["GET", "POST"])
+@login_required(return_app=SupportedApp.POST_AWARD_FRONTEND)
+@check_internal_user
+def retrieve_spreadsheet(fund_code: str, submission_id: str):
+    object_name = f"{fund_code}/{submission_id}"
+
+    try:
+        file_metadata = get_file_metadata(Config.AWS_S3_BUCKET_SUCCESSFUL_FILES, object_name, True)
```

Comment:
Sort of related to the comment about this function signature - but this floating `True` is hard to read because at a glance I have no idea what it means. If we keep this (which maybe I hope we don't), it'd probably be good to force it to be a [keyword-only argument](https://peps.python.org/pep-3102/#specification).

Reply:
:+1: